### PR TITLE
added cdf and ppf to jax.scipy.stats.uniform

### PR DESCRIPTION
**Pull request by IlayMenahem**
the PR is related to issue #13291

### CODE REVIEW

File: jax/_src/scipy/stats/uniform.py [hunk 1]
```diff
@@ -32,3 +33,22 @@ def logpdf(x: ArrayLike, loc: ArrayLike = 0, scale: ArrayLike = 1) -> Array:
...
+  vals = [zero, one, lax.div(lax.sub(x, loc), scale)]
+
+  return jnp.select(conds, vals)
+
+@_wraps(osp_stats.uniform.ppf, update_doc=False)
+def ppf(q: ArrayLike, loc: ArrayLike = 0, scale: ArrayLike = 1) -> Array:
+  q, loc, scale = promote_args_inexact("uniform.ppf", q, loc, scale)
+  zero, one = jnp.array(0, q.dtype), jnp.array(1, q.dtype)
+  return where(
+    jnp.isnan(q) | (q < zero) | (q > one),
```
Comment (jakevdp) [inline jax/_src/scipy/stats/uniform.py:51]: You can use `(q < 0) | (q > 1)` here, because this is not a context where strict dtype matching is required.

**Review by jakevdp — CHANGES_REQUESTED** [review]
Looks good - one small comment below. Also please squash your changes into a single commit (see https://jax.readthedocs.io/en/latest/contributing.html#single-change-commits-and-pull-requests)

**Review by jakevdp — APPROVED** [review]
Thanks!

**Review by jakevdp — CHANGES_REQUESTED** [review]

File: tests/scipy_stats_test.py [hunk 2]
```diff
@@ -1043,6 +1043,34 @@ def args_maker():
...
+    rng = jtu.rand_default(self.rng())
+    scipy_fun = osp_stats.uniform.cdf
+    lax_fun = lsp_stats.uniform.cdf
+
+    def args_maker():
+      x, loc, scale = map(rng, shapes, dtypes)
+      return [x, loc, np.abs(scale)]
+
+    with jtu.strict_promotion_if_dtypes_match(dtypes):
+      self._CheckAgainstNumpy(scipy_fun, lax_fun, args_maker,tol=1e-5)
```
[the remark this answers is not in the document]
Comment (IlayMenahem) [reply tests/scipy_stats_test.py]: right, i corrected it

**Review by jakevdp — APPROVED** [review]

**Pull request merged**